rfcs/0002-helm-oci/README.md

```diff
@@ -156,19 +156,25 @@ Bucket API design, where the same Kind servers different implementations: AWS S3
 
 In source-controller we'll add a new predicate for filtering `HelmRepositories` based on the `spec.type` field.
 
-The current `HelmRepositoryReconciler` will be renamed to `HelmRepositoryDefaultReconciler`,
-it's scope remains unchanged, and it will handle only objects with `type: Default`.
+The current `HelmRepositoryReconciler` will handle only objects with `type: default`,
+it's scope remains unchanged.
 
 We'll introduce a new reconciler named `HelmRepositoryOCIReconciler`, that will handle
-objects with `type: OCI`. This reconciler will set the `HelmRepository` Ready status to
+objects with `type: oci`. This reconciler will set the `HelmRepository` Ready status to
 `False` if the URL is not prefixed with `oci://`, otherwise the Ready status will be set to `True`.
```

Comment:
I think this should also mention that it verifies the credentials, which has an impact on status state.